FreeRTOS-Plus/Demo/FreeRTOS_IoT_Libraries/ota/common/DemoTasks/aws_iot_ota_update_demo.c

```diff
@@ -73,13 +74,51 @@ static void App_OTACompleteCallback( OTA_JobEvent_t eEvent );
  */
 static void prvInitialiseLibraries( void );
 
+/**
+ * @brief Delay before retrying network connection up to a maximum interval.
+ */
+static void _connectionRetryDelay( void );
+
+/**
+ * @brief Initialize the libraries required for OTA demo.
+ *
+ * @return `EXIT_SUCCESS` if all libraries were successfully initialized;
+ * `EXIT_FAILURE` otherwise.
+ */
+
+static void prvNetworkDisconnectCallback( void * param,
+										  IotMqttCallbackParam_t * mqttCallbackParams );
```

Comment:
indentations slightly off here and other function signatures.

Reply:
I think this is Github's problem, it's not correctly showing "tab", which is what we're using instead of spaces for demos. I did run uncrustify on this file, and it shows no change.